tests/Ormolu/FixitySpec.hs

```diff
@@ -261,7 +261,7 @@ checkFixities dependencies fixityImports expectedResult =
   where
     actualResult =
       fmap
-        (\(k, _) -> (k, inferFixity k resultMap))
+        (\(k, _) -> (k, inferFixity False k resultMap))
```

Comment:
Call sites like this would make it nice if we used some dedicated type instead of `Bool`, eg
```haskell
data DebugLogging = DebugLoggingEnabled | DebugLoggingDisabled
```
(unless this is too annoying, but I don't immediately see a big obstacle)

Reply:
Yeah, but then in general it applies to pretty much any `Bool` and we have quite a few of them, see `Config` for example. Maybe a separate PR could be opened for this kind of refactoring.

Reply:
Do you have an opinion whether [`choice`](https://github.com/mboes/choice/#choice-a-solution-to-boolean-blindness) might be worth it? I think it could be a nice as an alternative to our various "blind `Bool`s" (and even some ad-hoc two-constructor types, like `IsApplicand` or `ForAllVisibility`).

Reply:
`choice` looks nice!